roles/cdi/tasks/provision.yml

```diff
@@ -15,48 +15,29 @@
   when: ns.rc != 0
         and platform == "openshift"
 
+- name: Render CDI operator
+  template:
+    src: "cdi-operator.yaml.j2"
+    dest: "/tmp/cdi-operator.yaml"
+
+- name: Create CDI Operator
+  command: "{{ cluster_command }} apply -f /tmp/cdi-operator.yaml --validate=false"
+
 - name: Render {{ cdi_namespace }} ResourceQuota deployment yaml
   template:
-    src:  cdi-resourcequota.yml
+    src: cdi-resourcequota.yml.j2
     dest: /tmp/cdi-provision-resourcequota.yml
 
 - name: Create {{ cdi_namespace }} ResourceQuota
   command: "{{ cluster_command }} apply -f /tmp/cdi-provision-resourcequota.yml -n {{ cdi_namespace }}"
 
-- name: Check for cdi-controller.yaml.j2 template in {{ cdi_template_dir }}
-  stat:
-    path: "{{ cdi_template_dir }}/cdi-controller.yaml.j2"
-  register: byo_template
-
-- name: Render CDI deployment yaml from template in {{ cdi_template_dir }}
+- name: Render CDI operator resources
   template:
-    src: "cdi-controller.yaml.j2"
-    dest: "/tmp/cdi-controller.yml"
-  when: byo_template.stat.exists == true
-
-- name: Get CDI from {{ cdi_release_manifest_url }}
-  block:
-  - name: Check for cdi-controller.yaml.j2 template in /tmp
-    stat:
-      path: "/tmp/cdi-controller.yaml.j2"
-    register: downloaded_template
-
-  - name: Download CDI Template
-    get_url:
-      url: "{{ cdi_release_manifest_url }}/{{ release_tag }}/cdi-controller.yaml.j2"
-      dest: "/tmp/cdi-controller.yaml.j2"
-    when: downloaded_template.stat.exists == false
-
-  - name: Render CDI deployment yaml
-    template:
-      src: "/tmp/cdi-controller.yaml.j2"
-      dest: "/tmp/cdi-controller.yaml"
-
-  when: byo_template.stat.exists == false
-
-- name: Create CDI deployment
-  command: "{{ cluster_command }} apply -f /tmp/cdi-controller.yaml"
-
-- name: Enable privileged containers in the security context
-  command: "oc adm policy add-scc-to-user privileged -z cdi-sa -n {{ cdi_namespace }}"
-  when: platform == "openshift"
+    src: "cdi-operator-cr.yaml.j2"
+    dest: "/tmp/cdi-operator-cr.yaml"
+
+- name: Create CDI operator resources
+  command: "{{ cluster_command }} apply -f /tmp/cdi-operator-cr.yaml --validate=false"
+
+- name: Wait for CDI operator to report that everything is running
+  command: "{{ cluster_command }} -n {{ cdi_namespace }} wait cdis.cdi.kubevirt.io/cdi --for condition=available"
```

Comment:
pretty sure this command should be:  `wait cdis.cdi.kubevirt.io/cdi --for=condition=running`  

Also may want to add --timeout like 

`wait cdis.cdi.kubevirt.io/cdi --for=condition=running --timeout=120s`

Reply:
Thansk!